packages/scandipwa/src/component/MyAccountDownloadable/MyAccountDownloadable.style.scss

```diff
@@ -0,0 +1,18 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+@include mobile {
+    .MyDownloadableOrders {
+        overflow-x: scroll;
+        -webkit-overflow-scrolling: touch;
+        width: calc(100vw - 24px);
+    }
+}
```

Comment:
Move the include of mixing inside the block please. Because when we need to add common or desktop styles, we will do it anyway.